### ISSUE TITLE
Cannot run push jobs on nodes from Windows workstation

### ISSUE THREAD

**Issue by tpetchel** [chef-cft/wombat#108]
```
C:\Users\Administrator\cookbooks\awesome_customers_delivery [master ≡]> knife job start 'chef-client' --search 'recipe:awesome_customers_delivery'
WARNING: Falling back to Push Jobs v1 mode.
ERROR: You authenticated successfully to https://chef-server.chordata.biz/organizations/diprotodontia as workstation but you are not authorized for this action
Response:  User or client 'workstation' does not have access to that action on this server.
```

**Comment by tpetchel** [comment]
I needed to add the `workstation` user to the `admins` group. Can we add this to the automation?

**Comment by andrewelizondo** [comment]
That should be caught by this [line](https://github.com/chef-cft/wombat/blob/master/cookbooks/chef-server/recipes/cheffish.rb#L73), it may have caused a regression when we changed this to a node attribute.

**Comment by andrewelizondo** [comment]
This should be fixed by #112
